src/main/frontend/handler/events.cljs

```diff
@@ -272,10 +272,10 @@
     (fn [close-fn]
       [:div
        [:p
-        "Grant native filesystem permission for directory: "
```

Comment:
@sprocketc @queeup Would this word order be ok in other languages? I'm wondering if we should make this a fn translation just to be safe

Reply:
I always like to see whole sentence in one ```fn```.

Reply:
I agree that a function would be safer.

Reply:
@sprocketc @queeup Sounds good

@greatbody Can you add a fn translation for this? You'll need to pass in an argument to the fn translation e.g. `(t :settings-permission/grant-permission (config/get-local-dir repo))` and then in the translation - `(fn [dir] [:p "Grant ..." [:b dir]])` . I would normally just help your PR but your PR doesn't allow edits. See #6 in https://github.com/logseq/logseq/blob/master/CONTRIBUTING.md#pr-guidelines to allow edits in the future